Sources/_TestingInternals/include/Stubs.h

```diff
@@ -121,6 +121,24 @@ static int swt_siginfo_t_si_status(const siginfo_t *siginfo) {
 }
 #endif
 
+//#if defined(__wasi__)
+/// Get the version of the C standard library and runtime used by WASI, if
+/// available.
+///
+/// This function is provided because `WASI_LIBC_VERSION` may or may not be
+/// defined and may or may not be a complex macro.
+///
+/// For more information about the `WASI_LIBC_VERSION` macro, see
+/// [wasi-libc-#490](https://github.com/WebAssembly/wasi-libc/issues/490).
+static const char *_Nullable swt_getWASIVersion(void) {
+#if defined(WASI_LIBC_VERSION)
+  return WASI_LIBC_VERSION;
```

Comment:
As far as I checked, wasi-libc does not provide the version definition, unfortunately :cry:

Reply:
Not yet, but see the discussion in the issue linked above. :)

Reply:
Ok, let's hope they add it.

Reply:
Fingers crossed! If you know of any other version info we might use, I'm happy to take a PR here. :)